src/interpreter/built_ins.rs

```diff
@@ -47,7 +53,9 @@ mod chr {
         let i: i32 = interpreter.pop_integer();
         let mut s: String = String::new();
         s.push((i as u8) as char);
-        interpreter.function_result = s.into();
+        interpreter
```

Comment:
add a trait in interpreter that allows to set the variable in more ways

e.g. `interpreter.set_variable(resolved_declared_name, variant)`
`interpreter.set_variable(built_in_function, x where x into variant)`